src/error.rs

```diff
@@ -8,46 +8,35 @@ pub type Result<T, E = Error> = core::result::Result<T, E>;
 /// Error type of crypto.rs
 #[derive(Debug, PartialEq, Eq)]
 pub enum Error {
+    #[cfg(feature = "age")]
+    /// Age Format Error
+    AgeFormatError { inner: crate::keys::age::DecError },
```

Comment:
Just in case: you can use `AgeFormatError(crate::keys::age::DecError)`

Reply:
Maybe I should have

Reply:
I don't really mind, was just in case you were not aware of the syntax